docs/topics/db/models.txt

```diff
@@ -1343,11 +1343,33 @@ Field name "hiding" is not permitted
 -------------------------------------
 
 In normal Python class inheritance, it is permissible for a child class to
-override any attribute from the parent class. In Django, this is not permitted
-for attributes that are :class:`~django.db.models.Field` instances (at
-least, not at the moment). If a base class has a field called ``author``, you
-cannot create another model field called ``author`` in any class that inherits
-from that base class.
+override any attribute from the parent class. In Django, this is usually not
+permitted for attributes of :class:`~django.db.models.Model` classes that are
+:class:`~django.db.models.Field` instances. If a non abstract model base class
+has a field called ``author``, you cannot create another model field or
+define an attribute called ``author`` in any class that inherits from that base class.
+
+.. versionchanged:: 1.9
+
+    In earlier versions of django these restrictions applied to abstract base classes as well.
```

Comment:
earlier versions, (add comma + chop "of django")
please limit documentation lines to 79 characters.